shell_scripts/idm_postprocessing_periods_comparisons.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#SBATCH --output=sbatch_logs/postprocess_results/postprocess_tables_figures_period_comparison_%A.log
+#SBATCH --job-name=postprocess_period_comparisons
+#SBATCH --time=30-00:00
+#SBATCH --mem=8G
+#SBATCH -c 2
+#SBATCH --nodelist=idmodeling2
```

Comment:
TAXDIR is missing here so this won't work

Reply:
there's no TAXDIR in this script. i don't think we need it if it's not in the Rscript?